class.jetpack-autoupdate.php

```diff
@@ -35,9 +35,17 @@ private function __construct() {
 			add_filter( 'auto_update_theme',   array( $this, 'autoupdate_theme' ), 10, 2 );
 			add_filter( 'auto_update_core',    array( $this, 'autoupdate_core' ), 10, 2 );
 			add_action( 'automatic_updates_complete', array( $this, 'automatic_updates_complete' ), 999, 1 );
+
+			if ( Jetpack_Options::get_option( 'skip_version_control_check', false ) ) {
+				add_filter( 'automatic_updates_is_vcs_checkout', array( $this, '__return_false'), 99 );
```

Comment:
The reason why I went with array is because if for some reason another plugin or the user adds the __return false filter. I didn't want to remove the __return_false filter that another plugin put in place.